src/Toolkit/src/UxToolkitBundle.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Toolkit;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
+use Symfony\Component\HttpKernel\Bundle\AbstractBundle;
+use Symfony\UX\Toolkit\Command\BuildRegistryCommand;
+use Symfony\UX\Toolkit\Command\DebugUxToolkitCommand;
+use Symfony\UX\Toolkit\Command\UxToolkitInstallCommand;
+use Symfony\UX\Toolkit\Compiler\RegistryCompiler;
+use Symfony\UX\Toolkit\Compiler\TwigComponentCompiler;
+use Symfony\UX\Toolkit\ComponentRepository\CurrentTheme;
+use Symfony\UX\Toolkit\ComponentRepository\GithubRepository;
+use Symfony\UX\Toolkit\ComponentRepository\OfficialRepository;
+use Symfony\UX\Toolkit\ComponentRepository\RepositoryFactory;
+use Symfony\UX\Toolkit\ComponentRepository\RepositoryIdentifier;
+use Symfony\UX\Toolkit\DependencyInjection\ToolkitExtension;
+use Symfony\UX\Toolkit\Registry\DependenciesResolver;
+use Symfony\UX\Toolkit\Registry\Registry;
+use Symfony\UX\Toolkit\Registry\RegistryFactory;
+
+/**
+ * @author Jean-François Lépine
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+class UxToolkitBundle extends AbstractBundle
```

Comment:
```suggestion
class UXToolkitBundle extends AbstractBundle
```

due to the `UX` namespace, Flex assumes the bundle class should be `UXToolkitBundle`

Fixes install error of:

> Case mismatch between loaded and declared class names: "Symfony\UX\Toolkit\
>  UXToolkitBundle" vs "Symfony\UX\Toolkit\UxToolkitBundle"

because Flex added the `UX` version to my `bundles.php`